tests/menu/test_config.py

```diff
@@ -16,10 +15,11 @@ def test_view_configuration():
 
 
 def test_config_restore():
-    func_config_restore()
-    # removing backup file and testing of restore works
-    os.remove(BUILDTEST_CONFIG_BACKUP_FILE)
-    func_config_restore()
+
+    # removing config file and testing if reset works
+    os.remove(BUILDTEST_CONFIG_FILE)
+    func_config_reset()
+    assert True == os.path.exists(BUILDTEST_CONFIG_FILE)
```

Comment:
I think here you could just put
```python
assert(os.path.exists(BUILDTEST_CONFIG_FILE))
```

Reply:
good catch!

Reply:
Fixed now

Reply:
seems like black changes ``assert(os.path.exists(BUILDTEST_CONFIG_FILE))`` to ``assert os.path.exists(BUILDTEST_CONFIG_FILE)`` either way it still passes check